helm/templates/gw-secret.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.env.gateway.auth.enabled -}}
```

Comment:
add support for signature without auth

Reply:
auth is always needed
signature is needed in some cases